app/controllers/posts_controller.rb

```diff
@@ -4,8 +4,36 @@ def index
     @posts = Post.where(user_id: @user)
```

Comment:
@Reem-lab everything works well on my end, i want to believe you failed to create new users  in rails console when you ran the project, please do that and i think you should see everything come alive

Reply:
- [ ] ok, so I think it would be great if you created seeds in your side for your database and the reviewer would be easier to run the seed in his/her side because I am sure that I created two users but I couldn't be able to see them after running your project ✅

Reply:
@Reem-lab  you basically didnt create any user that is why you cannot see any information being displayed neither can you create posts or comments. however i will put in a couple of commands which you should use in rails c which will create two users and 4 posts as well as two comments each, and when you test all of the routes you will notice that they all work.
 `@first_user  = User.create(name: 'Tom', photo: 'https://unsplash.com
/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.', posts_counter: 
0).`
 **the posts_counter was missing from your initial creation thereby not saving any id for a user to the database** 
and this is how you create a post for that user `@first_post = Post.create(title: 'hello world', text: 'welcome', co
mments_counter: 0, likes_counter: 0, user: @first_user)`
and this is how you will create a comment as well `Comment.create(user: @first_user, post: @first_post, 
text: 'first comment')`

follow through all of the steps i have highlighted and you will see that everything works